### PR TITLE
Update missing description attributes for ITL CheckCommand definitions

### PR DESCRIPTION
**Pull request by ruzickap**

### CODE REVIEW

File: itl/command-plugins-manubulon.conf [hunk 1]
```diff
@@ -160,47 +187,61 @@ object CheckCommand "snmp-interface" {
...
 		}
-		"-d" = "$snmp_interface_delta$"
 		"-u" = {
 			set_if = "$snmp_interface_warncrit_percent$"
-		}
-		"-N" = {
-			set_if = "$snmp_interface_ifname$"
-		}
-		"-A" = {
-			set_if = "$snmp_interface_ifalias$"
```
Comment (dnsmichi) [inline itl/command-plugins-manubulon.conf]: Where did `-A` and `-N` go?

File: itl/command-plugins.conf [hunk 2]
```diff
@@ -1382,138 +1382,104 @@ object CheckCommand "disk" {
...
 			repeat_key = true
 			order = 1
 		}
-
-		"-p_old" = {
-			key = "-p"
-			value = "$disk_partition$"
-			order = 1
-		}
-
```
Comment (dnsmichi) [inline itl/command-plugins.conf]: Please leave them inside the configuration, there's a deprecation note in the docs, but it does not hurt for the time being.

File: itl/command-plugins.conf [hunk 3]
```diff
@@ -1382,138 +1382,104 @@ object CheckCommand "disk" {
...
 		"-x" = {
 			value = "$disk_partitions_excluded$"
 			description = "Ignore device (only works if -p unspecified)"
 		}
-
-		"-x_old" = {
-			key = "-x"
-			value = "$disk_partition_excluded$"
-		}
-
```
Comment (dnsmichi) [inline itl/command-plugins.conf]: Same as `disk_partition`, leave it inside the configuration.

**Comment by dnsmichi** [comment]
Thanks for your effort 👍  Please change the files from my review :)

File: itl/command-plugins-manubulon.conf (hunk 1 above)
Comment (ruzickap) [inline itl/command-plugins-manubulon.conf]: Actually there is no parameter "-A" and "-N" in http://nagios.manubulon.com/snmp_int.html
That's why I removed it...

Comment (dnsmichi) [inline itl/command-plugins-manubulon.conf]: Those were new additions I've accepted over at dnsmichi/manubulon-snmp. They are integrated inside the Debian package by @waja and to my knowledge no other distributions package these plugins. That is why the docs link to the patched repository and its v2.0 release.

Comment (ruzickap) [inline itl/command-plugins-manubulon.conf]: Oh... Ok I'll keep "-A" and "-N" then...

Comment (dnsmichi) [inline itl/command-plugins-manubulon.conf]: Cool thanks :)

**Comment by dnsmichi** [comment]
Thanks for the update, LGTM 👍

**Pull request merged**